docs/usage-examples/distinct.txt

```diff
@@ -0,0 +1,67 @@
+.. _laravel-distinct-usage:
+
+==============================
+Retrieve Distinct Field Values
+==============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: unique, different, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can retrieve distinct field values of documents in a collection by calling the ``distinct()``
+method on an object collection or a query builder.
+
+To retrieve distinct field values, pass a query filter to the ``where()`` method and a field name
+to the ``select()`` method. Then, call ``distinct()`` to return the unique field values of documents
+that match the query filter.
+
+Example
+-------
+
+This usage example performs the following actions:
+
+- Uses the ``Movie`` Eloquent model to represent the ``movies`` collection in the
+  ``sample_mflix`` database
+- Retrieves distinct field values of documents from the ``movies`` collection that match a query filter
+- Prints the distinct values
+
+The example calls the following methods on the ``Movie`` model:
+
+- ``where()``: matches documents in which the value of the ``year`` field is ``1930``.
+- ``select()``: retrieves the matching documents' ``imdb.rating`` field values.
```

Comment:
S: is there a better field to select for than a numerical field? Like genre or director maybe?

Reply:
Sure, changed to the directors field